src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -30,18 +32,114 @@ public sealed class MonitoringConfiguration : ConfigurationCategory
         public MonitoringConfiguration()
         {
             Snmp = new SnmpConfiguration();
+            OpenTelemetry = new OpenTelemetryConfiguration();
         }
 
         public SnmpConfiguration Snmp { get; }
+        public OpenTelemetryConfiguration OpenTelemetry { get; }
 
         public override void Initialize(IConfigurationRoot settings, HashSet<string> settingsNames, IConfigurationRoot serverWideSettings, HashSet<string> serverWideSettingsNames, ResourceType type, string resourceName)
         {
             base.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Snmp.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
-
+            OpenTelemetry.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Initialized = true;
         }
 
+        [ConfigurationCategory(ConfigurationCategoryType.Monitoring)]
+        public sealed class OpenTelemetryConfiguration : ConfigurationCategory
+        {
+            [Description("Indicates if OpenTelemetry is enabled or not. Default: false")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool Enabled { get; set; }
+            
+            [Description("Indicates if server-wide OpenTelemetry is enabled or not. Default: true")]
```

Comment:
How does it differ from the above option? I think we already talked about that and this option is supposed to enable / disable the server wide metrics / instruments, right? If yes then the description needs to be adjusted (config option name as well?)

Reply:
Well, when you want to expose dotnet instruments but don't want to expose server meters this could be useful.

Reply:
Below options has "... metrics are enabled or not". Note "metrics" while here you say "Open Telemetry is enabled"

Reply:
`Indicates if RavenDB's OpenTelemetry metrics are enabled or not. Default: true`?